camera-viewfinder-compose/src/main/java/com/google/jetpackcamera/viewfinder/CameraPreview.kt

```diff
@@ -41,13 +42,13 @@ import kotlinx.coroutines.flow.mapNotNull
 
 private const val TAG = "Preview"
 
+@SuppressLint("RestrictedApi")
 @Composable
 fun CameraPreview(
-    modifier: Modifier,
```

Comment:
We need to be able to pass in an optional the modifier for this one. It's a requirement for public Compose APIs

`modifier: Modifier = Modifier,`